setup.py

```diff
@@ -92,6 +92,6 @@ def run_tests(self):
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        'jsonpickle==1.4.2',
+        'jsonpickle==3.0.0',
```

Comment:
If no hard constraint is required, I would use ' jsonpickle>=3.3.0' to avoid future updates.

Reply:
I preferred to keep the authors choice of fixing the exactly version.